Composer/packages/intellisense/src/hooks/useLanguageServer.ts

```diff
@@ -70,7 +70,7 @@ export const useLanguageServer = (
 
   // Get completion results when selection changes
   React.useEffect(() => {
-    if (ws.current && ws.current.readyState === WebSocket.OPEN) {
+    if (ws.current && ws.current.readyState === WebSocket.OPEN && cursorPosition >= 0) {
```

Comment:
This probably deserves a comment about why the cursor position could be negative..